src/main/java/com/google/codeu/data/Guides.java

```diff
@@ -0,0 +1,65 @@
+package com.google.codeu.data;
+import java.util.UUID;
+
+public class Guides
+{
+	private UUID id;
+    private String name;
+    private String address;
+    private double contact_no;
+    private String gender;
+    private String location;
+    private double charge;
+    private long timestamp;
+
+    /**
+     * Constructs a new {@link Message} posted by {@code user} with {@code text} content. Generates a
+     * random ID and uses the current system time for the creation time.
+     */
+    public Guides(String name, String address, double contact_no, String gender, String location, double charge ) {
```

Comment:
A function with so many arguments is difficult to read. For constructors, it’s recommended to use builder pattern instead: https://www.geeksforgeeks.org/builder-pattern-in-java/